week3/World-Cup/components/Tournament.jsx

```diff
@@ -0,0 +1,189 @@
+import styled from "styled-components";
+import React, { useState } from "react";
+import FirstStage from "./FirstStage";
+import SecondStage from "./SecondStage";
+import ThirdStage from "./ThirdStage";
+import DRINK_LIST from "../src/assets/data";
+import TournamentResult from "./TournamentResult";
+
+const Tournament = () => {
+  const [tournamentStage, setTournamentStage] = useState(0);
+  const [firstChoice, setFirstChoice] = useState("");
+  const [secondChoice, setSecondChoice] = useState("");
+  const [thirdChoice, setThirdChoice] = useState("");
+
+  const filterList = () => {
+    const firstFilteredList = DRINK_LIST.filter(
+      (item) => item.category[0] === firstChoice
+    );
+
+    const secondFilteredList = firstFilteredList.filter(
+      (item) => item.category[1] === secondChoice
+    );
+
+    const thirdFilteredList = secondFilteredList.filter(
+      (item) => item.category[2] === thirdChoice
+    );
+
+    const finalResult = thirdFilteredList[0];
+
+    return finalResult;
+  };
```

Comment:
```suggestion
 const filterList = () => {
 let filteredList = DRINK_LIST;
 const choices = [firstChoice, secondChoice, thirdChoice];

 choices.forEach((choice, i) => {
   filteredList = filteredList.filter((item) => item.category[i] === choice);
 });

 return filteredList;
};
```

지금 상수 list에 있는 값 중에서 결과값과 일치하는 조건을 가진 값들을 빼오는 과정인 것 같은데
똑같은 비교 과정을 반복해서 써주는 것보다 `forEach`를 사용해서 이렇게 작성해주면 더 간결해 보이지 않을까요?ㅎㅎ,,